libbeat/processors/add_host_metadata/_meta/fields.yml

```diff
@@ -31,3 +31,10 @@
           type: keyword
           description: >
             OS family (e.g. redhat, debian, freebsd, windows).
+        - name: net.ip
+          description: >
+            List of IP-addresses.
+        - name: net.hw
+          description: >
+            List of hardware-addresses, usually MAC-addresses.
```

Comment:
I assume the type here would be `keyword`.

Reply:
Ok, I'll fix that